04_numeronym/joel00wood/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func main() {
+	fmt.Println(numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronym(s string) string {
+	var out string
+	len := len(s)
```

Comment:
stylistic / nitpick: I tend to avoid reusing names like this (`len`) - smart compilers can work out needs from context, but it can still tack on a little extra cognitive load when reading long code

and hypothetically, if `len()` returned a function - what would be behaviour of future calls (in scope) to it after this line?

Reply:
I'd imagine they'd return the value of len as set here and my unfortunate shortcut would have a less than desirable result, as such I've switched up for `sLen`.